frontend/src/components/7.Result/ResultPage.js

```diff
@@ -1,41 +1,41 @@
-import React, { useState, useEffect, useContext } from "react";
-import Help from "../Common/Help";
-import "../Common/Help.css";
+import React, { useState, useEffect, useContext } from 'react';
+import Help from '../Common/Help';
+import '../Common/Help.css';
 
-import Icon from "../Common/MapsPinpoint";
+import Icon from '../Common/MapsPinpoint';
 
-import axios from "axios";
+import axios from 'axios';
 
-import { SocketContext } from "../../sockets/SocketContext";
-import { Container, Row, Col } from "react-bootstrap";
-import Card from "react-bootstrap/Card";
-import Button from "react-bootstrap/Button";
-import SwipeCard from "../Common/SwipeCard";
+import { SocketContext } from '../../sockets/SocketContext';
+import { Container, Row, Col } from 'react-bootstrap';
+import Card from 'react-bootstrap/Card';
+import Button from 'react-bootstrap/Button';
+import SwipeCard from '../Common/SwipeCard';
 
-import MapModal from "../Common/MapModal";
-import "./ResultPage.css";
+import MapModal from '../Common/MapModal';
+import './ResultPage.css';
 
-import DataVisual from "./DataVisual";
-import Result from "./ResultPopup";
+import DataVisual from './DataVisual';
+import Result from './ResultPopup';
 
 // Dummy data, should be retrieved by sockets
-const name = "Lonestar";
-const location = "Botany";
-const price = "$$$";
+const name = 'Lonestar';
+const location = 'Botany';
+const price = '$$$';
 const rating = 4.0;
 
 /**
  * @param {*} props
  * @return {*}
  * TODO: remove hard-coded location for the winning restaurant coordinates
```

Comment:
this was left in from the older version before I added my comment,
What should we do about this?

Reply:
good catch! I've updated the file to remove this comment and another unneeded one.